lms/static/coffee/src/instructor_dashboard/util.coffee

```diff
@@ -206,31 +227,62 @@ create_email_message_views = ($messages_wrapper, emails) ->
 
     # Create hidden section for modal window
     email_id = email_info.email['id']
-    $message_content = $('<section>', "aria-hidden": "true", class: "modal email-modal", id: "email_message_" + email_id)
+    $message_content = $(
+      '<section>',
+      "aria-hidden": "true",
+      class: "modal email-modal",
+      id: edx.HtmlUtils.interpolateHtml("email_message_{id}", {id: email_id})
+    )
     $email_wrapper = $ '<div>', class: 'inner-wrapper email-content-wrapper'
     $email_header = $ '<div>', class: 'email-content-header'
 
     # Add copy email body button
-    $email_header.append $('<input>', type: "button", name: "copy-email-body-text", value: gettext("Copy Email To Editor"), id: "copy_email_" + email_id)
+    $email_header.append $(
+      '<input>',
+      type: "button",
+      name: "copy-email-body-text",
+      value: gettext("Copy Email To Editor"),
+      id: edx.HtmlUtils.interpolateHtml("email_message_{id}", {id: email_id})
+    )
 
     $close_button = $ '<a>', href: '#', class: "close-modal"
     $close_button.append $ '<i>', class: 'icon fa fa-times'
     $email_header.append $close_button
 
-    # HTML escape the subject line
-    subject_text = $('<span>').text(email_info.email['subject']).html()
-    $email_header.append $('<h2>', class: "message-bold").html('<em>' + gettext('Subject:') + '</em> ' + subject_text)
-    $email_header.append $('<h2>', class: "message-bold").html('<em>' + gettext('Sent By:') + '</em> ' + email_info.requester)
-    $email_header.append $('<h2>', class: "message-bold").html('<em>' + gettext('Time Sent:') + '</em> ' + email_info.created)
-    $email_header.append $('<h2>', class: "message-bold").html('<em>' + gettext('Sent To:') + '</em> ' + email_info.sent_to.join(", "))
+    # HTML escape things
+    interpolate_header = (title, value) ->
+      edx.HtmlUtils.setHtml(
+        $('<h2>', class: 'message-bold'),
+        edx.HtmlUtils.joinHtml(
+          edx.HtmlUtils.HTML('<em>'),
+          title
+          edx.HtmlUtils.HTML('</em>'),
+          value,
+        )
+      )
+    $subject = interpolate_header(gettext('Subject:'), email_info.email['subject'])
+    $requester = interpolate_header(gettext('Sent By:'), email_info.requester)
+    $created = interpolate_header(gettext('Time Sent:'), email_info.created)
+    $sent_to = interpolate_header(gettext('Sent To:'), email_info.sent_to.join(", "))
+    $email_header.append $subject
+    $email_header.append $requester
+    $email_header.append $created
+    $email_header.append $sent_to
```

Comment:
It's possible that the linter may fail on this in the future because it might be too painful to handle.  Could you switch it to the following while we are here and testing this?:
```
$email_header.append($subject)
```